packages/tree-widget/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@bentley/tree-widget-react",
+  "version": "2.0.0",
+  "description": "Tree Widget React",
+  "license": "MIT",
+  "main": "lib/tree-widget-react.js",
+  "typings": "lib/tree-widget-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets",
+    "clean": "rimraf lib build",
+    "lint": "tslint -p . 1>&2",
+    "test": "",
+    "cover": ""
+  },
+  "keywords": [
+    "Bentley",
+    "iModel Select React",
+    "iModel.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/visibility-widget"
+  },
+  "peerDependencies": {
+    "@bentley/bentleyjs-core": "^2.2.0",
```

Comment:
Is it a requirement to have this at `2.2.0` or is there a lower version that would suffice?

Reply:
2.2.0 introduces grouping elements by class in the model tree which IT3 makes use of and gates behind a feature flag.